bitnami/milvus/templates/_helpers.tpl

```diff
@@ -980,10 +980,33 @@ Init container definition for waiting for the database to be ready
 Init container definition for waiting for the database to be ready
 */}}
 {{- define "milvus.prepareMilvusInitContainer" -}}
+- name: copy-default-configuration
+  image: {{ template "milvus.image" .context }}
+  imagePullPolicy: {{ .context.Values.milvus.image.pullPolicy }}
+  {{- $block := index .context.Values .component }}
+  {{- if $block.containerSecurityContext.enabled }}
+  securityContext: {{- include "common.compatibility.renderSecurityContext" (dict "secContext" $block.containerSecurityContext "context" .context) | nindent 4 }}
+  {{- end }}
+  {{- if $block.resources }}
+  resources: {{- toYaml $block.resources | nindent 4 }}
+  {{- else if ne $block.resourcesPreset "none" }}
+  resources: {{- include "common.resources.preset" (dict "type" $block.resourcesPreset) | nindent 4 }}
+  {{- end }}
+  command:
+    - bash
+    - -ec
+    - |
+      #!/bin/bash
+      echo "Copying milvus default configuration"
+      cp -r /opt/bitnami/milvus/configs/. /bitnami/milvus/input-conf
```

Comment:
Why are we copying twice the default config? From `/opt/bitnami/milvus/configs/` to `/bitnami/milvus/input-conf`, then from `/bitnami/milvus/input-conf` to `/bitnami/milvus/rendered-conf`